.travis.yml

```diff
@@ -30,6 +30,8 @@ script:
   - ./run-qa-checks
   - coverage run --source=openwisp_users runtests.py
   - SAMPLE_APP=1 coverage run --append --source=openwisp_users runtests.py
+  # - pip install https://github.com/openwisp/openwisp-utils/archive/master.zip
+  - pip install https://github.com/atb00ker/openwisp-utils/archive/issues/106-add-default-menu-func.zip
```

Comment:
This can now be updated to `pip install -U https://github.com/openwisp/openwisp-utils/tarball/master`